tests/profiling/collector/test_stack.py

```diff
@@ -26,7 +26,7 @@
 # Python 3.11.9 is not compatible with gevent, https://github.com/python/cpython/issues/117983
 # The fix was not backported to 3.11. 3.12 got the fix but was not released yet.
 # Revisit this when 3.12.5 is released to check whether tests can be enabled.
-TESTING_GEVENT = os.getenv("DD_PROFILE_TEST_GEVENT", False) and sys.version_info < (3, 11, 9)
+TESTING_GEVENT = os.getenv("DD_PROFILE_TEST_GEVENT", False) and sys.version[:2] != (3, 11)
```

Comment:
Don't we need something like:
```suggestion
TESTING_GEVENT = os.getenv("DD_PROFILE_TEST_GEVENT", False) and sys.version[:2] < (3, 12)
```
to avoid all previous versions of Python (3.10, etc)?

Reply:
I just made a nonsensical change here. I'll update the comment and code soon.

Reply:
Please see updated.